docs/using-airbyte/core-concepts/sync-modes/full-refresh-overwrite.md

```diff
@@ -4,7 +4,7 @@
 
 The **Full Refresh** modes are the simplest methods that Airbyte uses to sync data, as they always retrieve all available information requested from the source, regardless of whether it has been synced before. This contrasts with [**Incremental sync**](./incremental-append.md), which does not sync data that has already been synced before.
 
-In the **Overwrite** variant, new syncs will destroy all data in the existing destination table and then pull the new data in. Therefore, data that has been removed from the source after an old sync will be deleted in the destination table.
+In the **Overwrite** variant, new syncs will destroy all data in the existing destination table and then pull the new data in. Therefore, data that has been removed from the source after an old sync will be deleted in the destination table.  
```

Comment:
```suggestion
In the **Overwrite** variant, new syncs will delete the existing table and erase all data in the existing destination table and then pull the new data again. Therefore, data that has been removed from the source after an old sync will be deleted in the destination table.  
```

Maybe make clear it will delete the table. I know it is not always the case but the design Airbyte thinks should like this correcT?

Reply:
This needs to apply to syncs that aren't going to databases (files, etc)